### PR TITLE
Add documentation cookbook for installing redirection.io

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -389,6 +389,90 @@ MERCURE_JWT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjc
...
+Finally, you need to edit
+`infrastructure/docker/services/frontend/etc/nginx/nginx.conf` to add the
+following configuration in the `server` block:
+
+```
+redirectionio_pass redirectionio-agent:10301;
+redirectionio_project_key "AAAAAAAAAAAAAAAA:BBBBBBBBBBBBBBBB";
+```
+
+**Don't forget to change the projectè key**.
```
Comment (pyrech) [inline README.md:472]:
```suggestion
**Don't forget to change the project key**.
```

File: README.md [hunk 2]
```diff
@@ -389,6 +389,90 @@ MERCURE_JWT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjc
...
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+```
+
+You can group this command with another one.
+
+Then, **after** installing nginx, you need to install the the module:
```
Comment (pyrech) [inline README.md:452]:
```suggestion
Then, **after** installing nginx, you need to install the module:
```

File: README.md (hunk 1 above)
Comment (welcoMattic) [inline README.md:472]:
```suggestion
**Don't forget to change the project key**.
```

**Review by welcoMattic — APPROVED** [review]

**Comment by lyrixx** [comment]
Thanks for the feedback. I applied them manually + simplified Dockerfile

**Pull request merged**